src/VCR/Util/StreamProcessor.php

```diff
@@ -171,6 +171,12 @@ protected function shouldProcess($uri)
      */
     public function stream_open($path, $mode, $options, &$openedPath)
     {
+        $subMode = substr($mode, 0, 1);
+
+        if (in_array($subMode, array('r', 'w', 'a')) && !is_file($path)) {
```

Comment:
I'm not sure this is correct now because according to the [documentation of fopen()](http://php.net/manual/en/function.fopen.php) all modes will create the file if it doesn't exist except `r` and `r+`. But I'm not an expert on this topic :)

Reply:
Oh, actually the "a" is an error, sorry. r and w don't create files

Reply:
To verify that you can add the other modes on the dataProvider in the test and see the file is created

Reply:
Ah, ok, but for `w` it also says "If the file does not exist, attempt to create it". Is the documentation wrong then?

Reply:
Indeed, I lost this last sentence in the docs, and I even tried to double check every option :/